platform-enterprise_versioned_docs/version-25.2/studios/add-studio.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Add a Studio"
+description: "Add a Studio in Platform."
+date created: "2025-09-04"
+last updated: "2025-09-04"
+tags: [data, session, studios]
+---
+
+Select the **Studios** tab in Platform to add a new Studio.
+
+:::note
+- If you're not able to see the Studios tab, contact your Platform adminstrator.
+- Review the user roles documentation for details about role permissions.
+- The following functionality is available to users with the **Maintain** role and above.
+:::
+
+Users with the **Maintain** role and above can add Studios via the **Studios** tab in your workspace. Select **Add Studio** to get started.
+
+## Compute and Data
+
+- **Resource labels**: Any [resource label](../labels/overview) already defined for the compute environment is added by default. adfdAdditional custom resource labels can be added or removed as needed.
+- **CPUs allocated**: The default allocation is 2 CPUs.
+- **GPUs allocated**: Available only if the selected compute environment has GPU support enabled. For more information about GPUs on AWS, see [Amazon ECS task definitions for GPU workloads][aws-gpu]. The default allocation is 0 GPUs.
+- **Maximum memory allocated**: The default allocation is 8192 MiB of memory.
+   
+### Mount data
+
+Select **Mount data**, and then from the **Mount data** modal, select data to mount. Select **Mount data** to confirm your selection.
+
+Datasets are mounted using the [Fusion file system](https://docs.seqera.io/fusion) and are available at `/workspace/data/<dataset>`. Mounted data doesn't need to match the compute environment or region of the cloud provider of the Studio. However, this might cause increased costs or errors.
```

Comment:
```suggestion
Data repositories are mounted using the [Fusion file system](https://docs.seqera.io/fusion) and are available at `/workspace/data/<data_repository>`. Mounted data doesn't need to match the compute environment or region of the cloud provider of the Studio. However, this might cause increased costs or errors.
```